pkg/controller/kieapp/constants/constants.go

```diff
@@ -260,6 +260,8 @@ const (
 	//Envs
 	KubeNS     = "KUBERNETES_NAMESPACE"
 	KubeLabels = "KUBERNETES_LABELS"
+
+	JBPMCluster = "JBPM_CLUSTER"
```

Comment:
this doesn't appear to be used anywhere in the code... is it necessary?

Reply:
No, was used when the PR was based on env var, then Felipe asked to use UI to set the flag